modules/manage/partials/schema-reg-ui.adoc

```diff
@@ -0,0 +1,117 @@
+In the Redpanda UI, the **Schema Registry** tab lists registered and verified schemas, including their serialization format and versions. Select an individual schema to see which topics it applies to. 
+
+[NOTE]
+====
+The Schema Registry is built into Redpanda, and you can use it with the Schema Registry API or with the Redpanda UI. This section describes Schema Registry operations available in the Redpanda UI. See also:
+
+* xref:manage:schema-reg/schema-reg-overview.adoc[]
+* xref:manage:schema-reg/schema-reg-api.adoc[]
+====
+
+ifdef::env-console[]
```

Comment:
```suggestion
ifndef::env-cloud[]
```

Reply:
I wouldn't use Console as the environment. Instead we want this to be displayed in all contexts that aren't Redpanda Cloud.